plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestLegacySyntaxForIcebergSnapshotAccess.java

```diff
@@ -28,7 +28,7 @@
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-public class TestIcebergSnapshots
+public class TestLegacySyntaxForIcebergSnapshotAccess
```

Comment:
You can ssquash this together with `Allow configuring support for snapshot access with legacy syntax` into a commit name sth like `Deprecate Iceberg snapshot access using mangled table names`.